calicoctl/commands/datastore/migrate/export.go

```diff
@@ -297,6 +321,7 @@ Description:
 
 	// Use the v1 API in order to retrieve IPAM resources
 	ipam := NewMigrateIPAM(client)
+	ipam.SetNodeMap(etcdToKddNodeMap)
```

Comment:
Feels like this should just be another argument to the NewMigrateIPAM, given its required?

Reply:
I didn't add it as an argument since it was really only required for export. Since I reuse the object for import, I didn't want to have to pass in an empty mapping just for initialization purposes there.